src/operator/numpy/linalg/np_eigvals.cu

```diff
@@ -29,11 +29,19 @@ namespace mxnet {
 namespace op {
 
 NNVM_REGISTER_OP(_npi_eigvals)
+.set_attr<FIsCUDAGraphsCompatible>("FIsCUDAGraphsCompatible",
+    [](const NodeAttrs&, const bool) {
+      return false;
+    })
 .set_attr<FCompute>("FCompute<gpu>", EigvalsOpForward<gpu>);
 
 #if MXNET_USE_CUSOLVER == 1
 
 NNVM_REGISTER_OP(_npi_eigvalsh)
+.set_attr<FIsCUDAGraphsCompatible>("FIsCUDAGraphsCompatible",
```

Comment:
istead of setting false everywhere, can we just check if `hasAttr("FIsCUDAGraphsCompatible")` so that by default its false?

Reply:
Well, the "everywhere" here is actually way less than if we went the other way around (only a bunch of operators with `FCompute` has synchronization that is not allowed under graphs.

Reply:
And the long term goal here is actually to make those excluded operators compatible too.